step.sh

```diff
@@ -168,8 +168,12 @@ if [ -z "${firebase_token}" ] ; then
     fi
 fi
 
+if [ -n "${FIREBASE_TOKEN}" ]  && [ -n "${service_credentials_file}" ]; then
+    echo_warn "Both authentication methods are defined: Firebase Token (via FIREBASE_TOKEN environment variable) and Service Credentials Field, one is enough."
```

Comment:
hey, why do we have to use the uppercase one? I thought the lowercase one was already filled with the environment variable.

Reply:
We don't have to. We even shouldn't use it. However, it is still recognized by CLI and a lot of people just call the secrets like that - `FIREBASE_TOKEN`.